doc/packaging.md

```diff
@@ -327,7 +327,8 @@ Notes on behavior:
 
 ### Downgrading
 
-The old configuration from `iotedge` is not removed from its location by any of the above actions; therefore, downgrading from the new package to the old one involves uninstalling the `aziot-edge` and `aziot-identity-service` packages, then reinstalling the `iotedge` package.
+The old configuration from `iotedge` is not removed from its location by any of the above actions; therefore, downgrading from the new package to the old one involves uninstalling the `aziot-edge` and `aziot-identity-service` packages, then reinstalling the `iotedge` package. Deployed modules that use the workload API to encrypt/decrypt secrets will need to be re-deployed.
```

Comment:
Why do they need to be re-deployed?

Reply:
I supposed it depends on whether they're authored to gracefully recover from a failure to read any secret that was saved in the new format, no?

Reply:
2 clarifications here -
- re-deploying means the modules need to be removed and re-added. Applying the same deployment again won't help
- If the encrypted data was stored on a host mounted folder/volume, then re-deploying modules won't help